### PR TITLE
feat: Get compiled job assets from getJobRunInput API

### PR DESCRIPTION
**Pull request by Mryashbhardwaj**

### CODE REVIEW

File: odpf/optimus/core/v1beta1/job_run.proto [hunk 1]
```diff
@@ -98,13 +105,32 @@ message RegisterInstanceRequest {
...
+
+  string project_name = 1;
+  string job_name = 2;
+  google.protobuf.Timestamp scheduled_at = 4;
+
+  string instance_name = 5;
+  InstanceSpec.Type instance_type = 6;
+
+  // either set job_name if this is a scheduled execution
+  // or set jonrun_id if this is a manual triggered execution
```
Comment (sravankorumilli) [inline odpf/optimus/core/v1beta1/job_run.proto:119]: jobrun_id comment type

**Review by sravankorumilli — APPROVED** [review]

**Review by sravankorumilli — APPROVED** [review]

File: odpf/optimus/core/v1beta1/job_run.proto [hunk 2]
```diff
@@ -98,13 +105,32 @@ message RegisterInstanceRequest {
...
+
 message RegisterInstanceResponse {
   ProjectSpecification project = 1;
   NamespaceSpecification namespace = 4;
   JobSpecification job = 2;
 
   InstanceSpec instance = 3;
-  InstanceContext context = 5;
+  
+  reserved 5;
```
Comment (sravankorumilli) [inline odpf/optimus/core/v1beta1/job_run.proto:131]: if it  is just a rename lets not bother about backward compatibility imo as all the existing clients will still work.

**Review by sravankorumilli — CHANGES_REQUESTED** [review]

File: odpf/optimus/core/v1beta1/job_run.proto (hunk 1 above)
Comment (Mryashbhardwaj) [reply odpf/optimus/core/v1beta1/job_run.proto:119]: i could not understand this please

**Review by sravankorumilli — APPROVED** [review]

File: odpf/optimus/core/v1beta1/job_run.proto (hunk 1 above)
Comment (Mryashbhardwaj) [reply odpf/optimus/core/v1beta1/job_run.proto]: fixed the typo issue

**Pull request merged**